lib/super_parser/account.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'transactions'
+require_relative 'helper_methods'
+
+class Account
+  include HelperMethods
+
+  attr_accessor :name, :currency, :balance, :nature
+
+  def initialize(opts = {})
```

Comment:
You always set parameter in `Account` class 
```ruby
 Account.new(account_details)
```
No need to set default parameter `{}` in this method.  
```ruby
def initialize(opts)
```